test/scripts/lib/utils.py

```
@@ -0,0 +1,121 @@
+        return False
+
+def run_command(command, host, ssh_key='~/.ssh/id_rsa', ssh_user='root', output=None, message=None, check_output=False):
+    if host not in ['localhost','127.0.0.1','::1']:
+        ssh_connection = 'ssh %s -i %s -l %s -o StrictHostKeyChecking=no -- ' \
+                         % (host, ssh_key, ssh_user)
+        command = '{0} "{1}"'.format(ssh_connection, command)
+
+    if check_output is False:
+        try:
+           with open(output, 'a') as job_log:
+               cmd_pipe = subprocess.Popen(command, stdout=job_log, stderr=subprocess.STDOUT, shell=True)
```

Ruff (0.15.6)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)